### PR TITLE
printenv: add page

### PR DESCRIPTION
**Pull request by ckauf**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

This is for issue #2213

### CODE REVIEW

File: pages/linux/printenv.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+> Print values of all or specific environment variables.
+
+- Display key-value pairs of all environment variables:
+
+`printenv`
+
+- Display the value of a specific variable:
+
+`printenv {{USERNAME}}`
```
[the remark this answers is not in the document]
Comment (ckauf) [reply pages/linux/printenv.md]: I was torn between USERNAME and HOME. If HOME is more common I'm fine to use that instead.

File: pages/linux/printenv.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Display the value of a variable and end with NUL instead of newline:
+
+`printenv --null {{USERNAME}}`
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/printenv.md]: Erm good point! This is one of those awkward ones, like `ls` for example - where the short options are more commonly used than the long ones. I'm tempted to say go for the short one, but I'm curious to know what @pxgamer thinks.

File: pages/linux/printenv.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/printenv.md]: Yeah, I think that `HOME` is more common across multiple distributions. Perhaps we could use that instead?

File: pages/linux/printenv.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ckauf) [reply pages/linux/printenv.md]: Either way is okay with me

**Comment by sbrl** [comment]
@stellarhopper: Thanks for the reviews! If you submit 5 (non-trivial) PRs, then we can add you as a collaborator on this repository, as detailed in out [Community Roles document](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md). Then your reviews will count towards merging :smiley: :fireworks:

**Comment by stellarhopper** [comment]
@sbrl  Absolutely, I'll keep an eye out for things I can contribute!

File: pages/linux/printenv.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pxgamer) [reply pages/linux/printenv.md]: I feel that `--null` may be slightly more descriptive for new users. But at the same time, I'm more used to `-0`. I'm not sure what I would prefer if I was new to this tool.

**Comment by sbrl** [comment]
If you're looking for pages to write and you're a Linux person, then you might like #2213 and #2214. There's a good list in #1579 for Windows users, but as far as I can tell there's only 1(!) command left there - `wmic`.

**Pull request merged**